docs/send-data/collect-from-other-data-sources/collect-metrics-telegraf/telegraf-collection-architecture.md

```diff
@@ -4,96 +4,81 @@ title: Telegraf Collection Architecture
 description: Learn about using Telegraf in metric collection pipelines, inside Kubernetes and in non-Kubernetes environments.
 ---
 
+This topic describes how to configure Telegraf plugins to fetch metrics from your applications and send them to Sumo Logic.
 
+Telegraf is an open source, plugin-based collector/agent that uses built-in *input plugins* to fetch metrics from the managed application, service, or third-party API where Telegraf is running, and then uses *output plugins* to send those collected metrics to another system, such as Sumo Logic. The Sumo Logic output plugin sends the metrics collected by the input plugin to an HTTP Source running on a Sumo Logic Hosted Collector.
 
-This topic has information about using Telegraf to obtain metrics from an application and to send those metrics to Sumo Logic. 
+Telegraf is used broadly in monitoring pipelines because it supports collection from a large number of applications and services. Input and output plugins are application-specific; for example, there are input plugins for Redis, Nginx, JMX, and more, and output plugins for services like Kafka, Prometheus, and of course Sumo Logic. You can see lists of Telegraf’s built-in input plugins [here](https://github.com/influxdata/telegraf).
```

Comment:
```suggestion
Telegraf is a popular choice for monitoring pipelines because it supports collection from a large number of applications and services. Input and output plugins are application-specific; for example, there are input plugins for Redis, Nginx, JMX, and more, and output plugins for services like Kafka, Prometheus, and of course Sumo Logic. You can see lists of Telegraf’s built-in input plugins [here](https://github.com/influxdata/telegraf).
```